File: stellarplugin/lib/data_models/account_response_bag.dart
import 'package:stellarplugin/data_models/account_response.dart';

class AccountResponseBag {
  String secretSeed;
  AccountResponse accountResponse;

  AccountResponseBag({this.secretSeed, this.accountResponse});

  AccountResponseBag.fromJson(Map<String, dynamic> json) {
    secretSeed = json['secretSeed'];
    accountResponse = AccountResponse.fromJson(json['accountResponse']);
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['secretSeed'] = this.secretSeed;
    data['accountResponse'] = this.accountResponse.toJson();
    return data;
  }
}

File: stellarplugin/android/src/main/java/com/boha/stellarplugin/AccountResponseBag.java
package com.boha.stellarplugin;

import org.stellar.sdk.responses.AccountResponse;

public class AccountResponseBag {
    final String secretSeed;
    final AccountResponse accountResponse;

    public AccountResponseBag(String secretSeed, AccountResponse accountResponse) {
        this.secretSeed = secretSeed;
        this.accountResponse = accountResponse;
    }

    public String getSecretSeed() {
        return secretSeed;
    }

    public AccountResponse getAccountResponse() {
        return accountResponse;
    }
}
